src/workloads/scale/LargeIndexedIns.yml

```diff
@@ -0,0 +1,78 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload sends bursts of finds that use large $ins while running
+  a low rate update workload.  The workload causes high CPU load as the server
+  becomes bottlenecked on tcmalloc spinlocks during the find operations.
+  Improvements or regressions in this aspect of the allocator should
+  be measurable by the average CPU usage during this test as well as
+  the latency of the find operations.
+
+  In this workload, large arrays of random strings are generated to use
+  for the $in queries.  To avoid a CPU bottleneck on the workload client,
+  it uses a ^Once generator to generate the arrays once during initialization.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 20000
+
+Actors:
+- Name: Loader
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: largeins
+    CollectionCount: 1
+    Threads: 1
+    DocumentCount: 1000
+    BatchSize: 1000
+    Indexes:
+    - keys: {"hash" : 1, "key" : 1, "string1" : 1, "string2" : 1}
```

Comment:
fwiw you don't need quotes here - `keys: {hash: 1, key:1, string1:1, string2: 1}`
Consistency helps legibility if nothing else.

Reply:
Updated